seqr/views/status.py

```diff
@@ -43,11 +44,11 @@ def status_view(request):
         if resp.status >= 400:
             raise ValueError('Error {}: {}'.format(resp.status, resp.reason))
     except Exception as e:
-        dependent_services_ok = False
+        secondary_services_ok = False
         logger.error('Kibana connection error: {}'.format(str(e)))
```

Comment:
Don't you think it is better to change the log from an `error` to a `warning`?

Reply:
no I need to be alerted if these services are down, its really bad if it happens